services/py-api/src/utils.py

```diff
@@ -43,3 +45,37 @@ async def stop(self) -> None:
             raise RuntimeError("The AsyncClient has not been initialized!")
 
         await self._async_client.aclose()
+
+class JwtUtility:
+    class UserData(TypedDict):
+        id: int
+        name: str
+        email: str
+        is_admin: bool
+        email_verified: bool
+        team_id: int
+        created_at: str
+        updated_at: str
+         
+
+    @staticmethod
+    def encode(data: UserData) -> str:
+            return encode(data,key=environ["SECRET_KEY"])
+    
+    @staticmethod
+    def decode(token: str) -> Dict[str, Any]:
+     try:
+        decoded_payload=jwt.decode(token, SECRET_KEY, algorithms=["HS256"])
```

Comment:
Same thing with the naming here. change the method that you define yourself to decode_data

Reply:
Put the algorithm in the .env variable and get it from there so that is safer. We are not showing all our visitors of the public repository the type of algorithm we are using for the Jwt secret

Reply:
Should I import the Secret key and the Algorithm in the beginning in some way, as there are not defined and Pylance doesn't recognise them?

Reply:
Yeah absolutely, I think you can just go ahead and do that. Thank you Plamena! Also about the testing part, as Ivan suggested just write an integration test as he describes it!

Reply:
Fixed in be223527926be992bf6729e80ee043728b8fa1c5